net-misc/chrony/metadata.xml

```diff
@@ -23,21 +23,23 @@
 		な接続でも機能します。
 	</longdescription>
 	<use>
-		<flag name="cmdmon">Support for command and monitoring</flag>
+		<flag name="cmdmon">Enable support for command and monitoring</flag>
 		<flag name="debug">Get DEBUG_LOG output from chronyd when passing -dd parameter</flag>
 		<flag name="html">Install HTML documentation</flag>
 		<flag name="nettle">Use <pkg>dev-libs/nettle</pkg> for hash functions or nts</flag>
 		<flag name="nss">Use <pkg>dev-libs/nss</pkg> for hash functions</flag>
-		<flag name="ntp">Support for the Network Time Protocol (NTP)</flag>
```

Comment:
I don't think we need to add "enable" for everything, and this also adds several typos ("upport").